src/snapred/backend/data/LocalDataService.py

```diff
@@ -103,70 +100,33 @@ def _readInstrumentParameters(self) -> Dict[str, Any]:
             rawVanadiumCorrectionFileName=reductionParameters["rawVCorrFileName"],
             vanadiumFilePath=str(
                 self.instrumentConfig.calibrationDirectory
                 / "Powder"
-                / reductionParameters["stateId"]
+                / stateId
                 / reductionParameters["rawVCorrFileName"]
             ),
-            calibrationMaskFileName=reductionParameters.get("CalibrationMaskFilename"),
-            stateId=reductionParameters["stateId"],
-            tofBin=reductionParameters["tofBin"],
-            tofMax=reductionParameters["tofMax"],
-            tofMin=reductionParameters["tofMin"],
-            version=reductionParameters["version"],
-            wallclockTof=reductionParameters["wallClockTol"],
-            temporalProximity=None,
+            stateId=stateId,
+            tofBin=self.instrumentConfig.delTOverT / self.instrumentConfig.NBins,
+            tofMax=particleBounds.tof.maximum,
+            tofMin=particleBounds.tof.minimum,
         )  # TODO: fill with real value
 
-    def _readDiffractionCalibrant(self, runId: str) -> DiffractionCalibrant:
-        reductionParameters = self._readReductionParameters(runId)
-
-        return DiffractionCalibrant(
-            filename=reductionParameters["calFileName"],
-            runNumber=reductionParameters["CRun"][0],
-            name=reductionParameters.get("CalibrantName"),
-            diffCalPath=str(
-                self.instrumentConfig.calibrationDirectory
-                / "Powder"
-                / reductionParameters["stateId"]
-                / reductionParameters["calFileName"]
-            ),
-            latticeParameters=None,  # TODO: missing, reductionParameters['CalibrantLatticeParameters'],
-            reference=None,
-        )  # TODO: missing, reductionParameters['CalibrantReference'])
-
-    def _readNormalizationCalibrant(self, runId: str) -> NormalizationCalibrant:
-        reductionParameters = self._readReductionParameters(runId)
-        return NormalizationCalibrant(
-            numAnnuli=reductionParameters["NAnnul"],
-            numSlices=None,  # TODO: missing, reductionParameters['Nslice'],
-            attenuationCrossSection=reductionParameters["VAttenuationXSection"],
-            attenuationHeight=reductionParameters["VHeight"],
-            geometry=None,  # TODO: missing, reductionParameters['VGeometry'],
-            FWHM=reductionParameters["VFWHM"],
-            mask=reductionParameters["VMsk"],
-            material=None,  # TODO: missing,
-            peaks=reductionParameters["VPeaks"].split(","),
-            radius=reductionParameters["VRad"],
-            sampleNumberDensity=reductionParameters["VSampleNumberDensity"],
-            scatteringCrossSection=reductionParameters["VScatteringXSection"],
-            smoothPoints=reductionParameters["VSmoothPoints"],
-            calibrationState=None,
-        )  # TODO: missing, reductionParameters['VCalibState'])
```

Comment:
Why are these methods removed?

Reply:
They are not used anywhere except for the ReductionAlgorithm(s) and they are incorrect, refering to proto data and not data generated by SNAPRed.  [Consider the following](https://github.com/neutrons/SNAPRed/blob/next/src/snapred/backend/recipe/algorithm/AlignAndFocusReductionAlgorithm.py#L27), it refers to some arbitrary calibration.